docs/Revocation.md

```diff
@@ -0,0 +1,96 @@
+
+> **Note** that Verifiable Credential Submission endpoint(`/vc-submission`) is specifically for submitting individual credentials and is distinct from a Verifiable Presentation (VP) submission.
+
+- Once the verifier scans the QR code, it decodes the QR code using `pixel_pass_library` and post it to the Inji Verify backend (db).
+
+- A typical JSON-LD Revoked Verifiable Credential includes a field like this:
+
+    ```json
+    "credentialStatus": {
+    "id": "https://example.org/status/24#94567",
+    "type": "StatusList2021Entry",
+    "statusPurpose": "revocation",
+    "statusListIndex": "94567",
+    "statusListCredential": "https://example.org/status/24"
+    }
+    ```
+
+- Field-by-Field Explanation
+
+| Field                 | Meaning                                                                                                   |
+|-----------------------|-----------------------------------------------------------------------------------------------------------|
+| id                    | A unique URL identifier for this status entry.                                                             |
+| type                  | The method used for status checking — e.g., `BitstringStatusListEntry`.                                   |
+| statusPurpose         | Describes why this status exists — e.g., "revocation" or "suspension".                                     |
+| statusListIndex       | A numeric index (or bit position) representing this credential’s position in the status list.             |
+| statusListCredential  | The URI of the status list credential — a VC published by the issuer that contains a compressed bitstring. |
+
+### Submission Result:
+
+- Inji verify UI can fetch the result of the submission through APIs.
+
+- This API performs server-side verification of a Verifiable Credential (VC) to validate its integrity and authenticity. It executes checks such as cryptographic signature validation and ensures that the credential has not been altered or tampered with.
+
+### How it check for revocation
+
+> - **_Issuer_**: assigns each new credential a statusListIndex and puts that index into the VC's credentialStatus object.
+>
+> - **_Inji Verify Backend_**: retrieves the `statusListCredential`, decodes its encodedList, checks the bit at that index to see whether it's revoked/suspended.
+
+- The verification status returned can be **_SUCCESS_**, **_INVALID_**, **_EXPIRED_** or **_REVOKED_**.
+
+- If the Inji Verify Backend encounters any error while retrieving the `statusListCredential` or verifying the status, it will post the error back to the Verify UI along with an error description and `status code` as `500`.
+
+
+## 2. OPENID4VP
+
+### Verifiable Presentation Submission:
+
+- Inji Verify have API to submit Verifiable Presentation.
+- Once the wallet scans the QR code, it generates the VP token and the submission request, which are then posted to the Inji Verify backend.
```

Comment:
The phrasing “…posted to the Inji Verify backend” may be misleading, since in this context the VP token and submission request are formally submitted, not just posted. can we update this line to use “submitted to the Inji Verify backend” instead.